### ISSUE TITLE
 Training loss is decreasing while validation loss is NaN

### ISSUE THREAD

**Issue by Priyanshu999** [abhijitjadhav1998/Deepfake_detection_using_deep_learning#11]
I found that although the training loss is consistently decreasing, the validation loss remains as NaN (validation accuracy is constant after every epoch). Can you fix this? 
![image](https://user-images.githubusercontent.com/55920219/187090776-a448216b-7a99-4ce8-ad93-746dc0f6ebfd.png)

**Comment by abhijitjadhav1998** [comment]
Please debug the code. there seem to issue with gradient decent

**Comment by Mithoon278** [comment]
did the issue solved?
